docs-md/operate-and-deploy/installation/server-config/config-reference.md

```diff
@@ -314,6 +314,29 @@ statements.
     `ksql.persistence.ensure.value.is.struct` setting, because it has no
     concept of an outer record or structure.
 
+### ksql.query.pull.enable.standby.reads
+
+Config to enable/disable forwarding pull queries to standby hosts when the active is dead. This means that stale values may be returned 
+for these queries since standby hosts receive updates from the changelog topic (to which the active writes to) asynchronously.
+Turning on this configuration, effectively sacrifices consistency for higher availability. 
+
+Setting to `true` guarantees high availability for pull queries. If set to `false`, pull queries will fail when the active is dead and 
```

Comment:
By default, when we set this to true, with nothing else (no heartbeats, no lag), it will try active first, and fall back to standby, right?